feature_tests/dart/lib/src/lib.g.dart

```diff
@@ -125,12 +128,25 @@ final class _ResultDoubleVoid extends ffi.Struct {
 
   @ffi.Bool()
   external bool isOk;
+
+  
+  factory _ResultDoubleVoid.ok(double val) {
+    final struct = ffi.Struct.create<_ResultDoubleVoid>();
+    struct.isOk = true;
+    struct.union.ok = val;
+    return struct;
+  }
+  factory _ResultDoubleVoid.err() {
+    final struct = ffi.Struct.create<_ResultDoubleVoid>();
+    struct.isOk = false;
+    return struct;
+  }
 }
 
 final class _ResultInt32OpaqueUnion extends ffi.Union {
   @ffi.Int32()
-  external int ok;
 
+  external int ok;
   external ffi.Pointer<ffi.Opaque> err;
```

Comment:
there should not be a blank line between the annotation and the field, and there should be one between the fields

Reply:
Ah, found it